oep-template.rst

```diff
@@ -50,32 +54,20 @@ existing architecture or process is inadequate to address the problem that the
 OEP solves, or why adopting the best practice would significantly improve Open
 edX.
 
-OEP submissions without sufficiently clear and compelling motivation are
-unlikely to be accepted.
-
 Specification
 =============
 
 The specification describes the technical details of the Architecture, Best
-Practice, Process or Product Enhancement proposed by the OEP. If the proposal
-includes a new API, specify its syntax and semantics.
+Practice or Process proposed by the OEP. If the proposal includes a new API,
+specify its syntax and semantics.
 
 Rationale
 =========
 
 The rationale adds to the specification by describing the events or
-requirements that led the proposal, what influenced the design, and why
-particular design decisions were made. It should describe any alternate designs
-that were considered and identify any related work, for example, how the
-feature is supported in other systems.
-
-The rationale should provide evidence of consensus within the community and
```

Comment:
Is it worth keeping this paragraph, but as a suggestion, rather than a soft requirement? `The rational *could* provide evidence...`

Reply:
We currently have this in 3 different places:
* **References** section in the header
* In this **Rationale** section
* In the **Rejected Alternatives** section below

I found it confusing and repetitive.  For now, I kept the References header and removed the rest.  Where do folks think it should belong?

I felt the References header was good for reading that information quickly at-a-glance.  If authors/arbiters want to capture further detail, they can always do so.